### ISSUE TITLE
Implement safe Vale styles

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse-sensitive-terms#6]
In https://github.com/WordPress/openverse/pull/3065 we attempted to add Vale to the monorepo. However, because Vale relies on locally defining styles, its usage inherently introduces lists of sensitive terms into the repository. We've worked hard to avoid that, and we can do so by moving the Vale configuration into this repository, and then using a docker image to run Vale in pre-commit to check in CI.

To do that, we'll follow this procedure:

- [ ] WordPress/openverse#7
- [ ] WordPress/openverse#8
    - Copy the Vale styles into the image
    - Copy the vale.ini into the image
    - Set the configuration file path in CMD to the vale.ini (otherwise vale looks for it in the CWD)
- [ ] WordPress/openverse#9
- [ ] [Add a new pre-commit hook to the monorepo using the `docker_image` language that runs the `openverse-vale` image against our documentation.](https://github.com/WordPress/openverse/issues/3566)

**Comment by sarayourfriend** [comment]
@ngken0995 would you like me to assign you to this issue?

**Comment by ngken0995** [comment]
@sarayourfriend Yes, I would like to be assign to this issue. Looking forward to working on it.

**Comment by sarayourfriend** [comment]
Great! Assigned. If you comment on the other issues I can assign them to you as well, but it's not as important in this repository, not much happens in here usually :slightly_smiling_face: 

As for picking the styles, go with what you think is best. I was thinking more about the lists from alex especially and I'm sceptical we need to lint for violent slurs (for example). It's never been an issue that someone opened a PR with them in it and if they did we'd close the PR immediately for being in violation of the WordPress code of conduct anyway.
